benchmark/test_special_perf.py

```diff
@@ -73,3 +73,20 @@ def resolve_conj_arg(dtype, batch, size):
         sizes=SIZES,
     )
     bench.run()
+
+
+def test_multinomial_with_replacement():
+    def multinomial_args(dtype, batch, size):
+        dist = torch.rand((batch, size), dtype=dtype, device="cuda")
+        n_samples = 10000
+        return (dist, n_samples, True)
+
+    bench = Benchmark(
+        op_name="multinomial",
+        torch_op=torch.multinomial,
+        arg_func=multinomial_args,
+        dtypes=(torch.float16, torch.float32),
```

Comment:
why not run benchmark on bf16?

Reply:
Bfloat is not supported yet for torch.multinomial.